src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepDynamicContextTest.java

```diff
@@ -114,14 +119,17 @@ public class ExecutorStepDynamicContextTest {
         sessions.then(j -> {
             // Start up a build and then reboot and take the node offline
             assertEquals(0, j.jenkins.getLabel("ghost").getNodes().size()); // Make sure test impl is correctly deleted
-            assertNull(j.jenkins.getNode("ghost")); // Make sure test impl is correctly deleted
```

Comment:
Useless assertion, since the node would just have the label `ghost` but usually named `slave0` based on generation.